prime-router/docs/getting-started/standard-operating-procedures/observation-mapping-table.md

```diff
@@ -26,6 +24,10 @@ Other useful options:
 See the NMLS VSAC Support Center [here](https://www.nlm.nih.gov/vsac/support/usingvsac/vsacfhirapi.html) to request an
 API key.
 
+### Important Notes on Update
+- This utility cannot support new OIDS because they must be manually mapped to a condition code. 
+- This utility does not update AOE question mappings
```

Comment:
I like the addition of these two lines, but I don't understand why this is okay? Perhaps we need to ask James?